pkg/asset/installconfig/openstack/validation/platform_test.go

```diff
@@ -249,3 +250,96 @@ func TestOpenStackPlatformValidation(t *testing.T) {
 		})
 	}
 }
+
+func TestClusterOSImage(t *testing.T) {
+	cases := []struct {
+		name           string
+		platform       *openstack.Platform
+		cloudInfo      *CloudInfo
+		networking     *types.Networking
+		expectedError  bool
+		expectedErrMsg string // NOTE: this is a REGEXP
```

Comment:
`expectedError  bool` is just duplication of the requirement from `expectedErrMsg string`, if expectedErrMsg is not empty the error must match that regex else when empty there should be no error.

Reply:
removed it